modules/machine-config-overview.adoc

```diff
@@ -56,7 +56,7 @@ The kinds of components that MCO can change include:
 * Adding new users by using a machine config is not supported.
 ====
 * **kernelArguments**: Add arguments to the kernel command line when {product-title} nodes boot.
-* **kernelType**: Optionally identify a non-standard kernel to use instead of the standard kernel. Use `realtime` to use the RT kernel (for RAN). This is only supported on select platforms.
+* **kernelType**: Optionally identify a non-standard kernel to use instead of the standard kernel. Use `realtime` to use the RT kernel (for RAN). This is only supported on select platforms. Use the `64k-pages` parameter for 64k memory pages on AArch64 machines. This setting is exclusive to clusters that use 64-bit ARM architectures.
```

Comment:
maybe:

```
Use the `64k-pages` parameter to enable 64k memory pages for the machines in a cluster. This setting is only supported for machines with ARM architecture. 
```

Reply:
I prefer the "is exclusive" wording rather than "only supported", but im gonna see what peer review says too